test/test_api_responses.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+from unittest.mock import MagicMock, patch
+
+import pytest
+import titan_client
+from .test_params import test_params
+
+prefix = os.path.abspath(os.path.dirname(__file__))
```

Comment:
This is used in `test_stix_mappers.py` as well. Let's create a file `conftest.py` (it has some special meaning in pytest) and import from there.